### ISSUE TITLE
This document seems to be changed!

### ISSUE THREAD

**Issue by nader321** [MicrosoftDocs/sql-docs#9455]
Hello
First of all thanks for your precise documentation :)
It seems this document has been changed.
I have a copy of a text that is not in this page any more:

"The JDBC XA distributed transaction components are included in the SQL Server engine in SQL Server 2017 starting with cumulative update 16 and in SQL Server 2019, and can be enabled or disabled with a system stored procedure. To enable the required components to perform XA distributed transactions using the JDBC driver, execute the following stored procedure."

Even the text is not here the information is correct because my SQL 2019 works very well with XA transactions without sqljdbc_xa.dll.

My problem is that there is no sqljdbc_xa.dll on the computer to find the version of JDBC.

How do I find the version of SQLJDBC on SQL server?

Thanks in advance.

Nader Salas
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4b3d30fd-771a-03a3-84f6-6fcea65fa709
* Version Independent ID: 1edbe4e2-3636-8071-0d5a-c8ea4b40307e
* Content: [Understanding XA transactions - JDBC Driver for SQL Server](https://learn.microsoft.com/en-us/sql/connect/jdbc/understanding-xa-transactions?view=sql-server-ver15)
* Content Source: [docs/connect/jdbc/understanding-xa-transactions.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/connect/jdbc/understanding-xa-transactions.md)
* Service: **sql**
* Sub-service: **connectivity**
* GitHub Login: @David-Engel
* Microsoft Alias: **v-davidengel**

**Comment by David-Engel** [comment]
Looking back, I updated the article after you filed #8736. The note was removed in favor of SQL Server version-specific instructions starting at [Configuring the JDBC distributed transaction components](https://learn.microsoft.com/sql/connect/jdbc/understanding-xa-transactions#configuring-the-jdbc-distributed-transaction-components).

Reading the article again, I think the instructions are pretty clear without the note. But I added a similar one back to the sections [SQL Server 2017 CU15 and lower](https://learn.microsoft.com/en-us/sql/connect/jdbc/understanding-xa-transactions?view=sql-server-ver15#sql-server-2017-cu15-and-lower) and [Upgrading sqljdbc_xa.dll](https://learn.microsoft.com/en-us/sql/connect/jdbc/understanding-xa-transactions?view=sql-server-ver15#upgrading-sqljdbc_xadll) to make it clear what they apply to.

> My problem is that there is no sqljdbc_xa.dll on the computer to find the version of JDBC.

In addition to my previous statement, in case it wasn't clear, the version of the JDBC driver doesn't matter with regard to the server-side component. The server-side functions should be backwards compatible with all in-support versions of the JDBC driver.

Regards,
David
